### DISCUSSION TITLE
Thinking Cap #3

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#10] [Assignments]
## Prompt:
> Consider what are your ethical views when it comes to society and technology? How do your ethics empower civil society? 

As always, completing the [readings](https://github.com/albertkun/211A-ASIAAM-191A/tree/master/Week_03#readings) will be helpful in this thinking cap.

## Submission
Make a new comment below in at least 3 - 4 sentences and respond to any two students posts before class begins.

**Comment by abi-200** [comment]
My ethical views when it comes to technology and society generally fall into the category that without consideration for both, we cannot hope to have a comprehensive view of ethical use of cartography, mapping, and/or GIS - this relates to lecture with both social and technological reductionism, and I believe that falling too much onto one side prevents one from truly considering all ethical boundaries of mapping that would best allow one to give power back to communities. In my day-to-day life, I tend to notice a lot more technological reductionism amongst my peers, similar to the Marx reading.I often hear/read about people reverting Google/cell phones/social media as the reason why they can stay educated and informed about the world around them. I also hear a lot about how people cannot live without their latest smartphone, iPad, etc, and whenever a new piece of technology comes out, people tend to want it as more of a status symbol rather than because their old technology stopped working adequately. Just as in the Marx article as well, I have also seen people solely blame social media as the major cause of consumerism, mental health issues, etc - while I do not contest that these are major contributing factors due to surveillance capitalism, the ethics of any new technology, not just with social media, are not the sole factor. Ethics must consider both the straightforward and stated use of the technology (internalist view, as in the Crampton article) with the broader societal implications (externalist view). For example, as the Marx article stated, social media can be a way for people to discuss societal issues already present, but it does not cause the issues at hand already (ex. Distrust of politicians), and it also states that new technology is not the ethical solution to social problems such as income inequality and privacy breaching. I think it is very interesting how the article mentions how social media platforms with secure messaging and lack of algorithm placement, such as Whatsapp, can just as effectively contribute to misinformation spread as larger websites such as Facebook - in my own family/friend circle, Whatsapp is definitely used to spread misinformation, and some family members of mine consider the app a tool to spread information that they believe will help others.

The Cramptom article highlights a common defense I have heard with regards to surveillance capitalism, and that is with collecting data not at the individual level but at the aggregate level (ex. City blocks instead of individual households) - however, as the article states, even this data can be used to negatively impact those in these regions via circumstances beyond their control, such as with higher premiums. When it comes to issues such as basic safety and healthcare, which in my ethical views falls under the category of basic human rights, restricting access to such items and justifying this restriction with data information collection is unethical purely because it does the opposite of empower those that the data is collected about. The article also notes many times about how the purpose of geodemographics is partially to exert “control” over communities and “control” consumer behaviors - in my view, any use of geographic information that seeks to obtain external control over communities from outside the community itself is unethical because it strips power away from those who may need it more, and who are entitled to it. Thus, since my views of ethics includes the fact that ethically, data use must strive to help communities and let them keep their own power of their own data, it empowers communities by accepting that data is almost never completely neutral, and that the inherent biases in data should contribute to as much net positives for the affected communities rather than for larger databases, politics, and others who do not have the communities best interest at hand.

**Reply by eandrusenki** [reply]
I have issues with the implicit biases that humans have and then use technology in a negative way towards certain groups, regions, demographics, etc. However, I really liked what you said about hopefully allowing biases contribute positively towards communities and not towards politics of higher powers.
I also liked "without consideration for both, we cannot hope to have a comprehensive view of ethical use of cartography, mapping, and/or GIS" and I think it's important that we understand our own views of society and technology as well as ethics to make sure our work in cartography and GIS is ethical and empowering.

**Comment by edmondywen** [comment]
The core of my ethical views is based on the key principle that technology ought to be decentered while people should be prioritized. Tech culture has always felt very weird but this past week’s material has really helped materialize my thoughts into words. Narratives around issues involving technology typically seem to personify technology as some overarching force while papering over the people and systems involved. For instance: criticism of mystical social media algorithms not the people who profit from them and celebration of carbon sequestration without regards to the root cause of climate change. The Jacobin article offers solid insight on this issue by arguing that The Social Dilemma “blames a whole range of problems on technology” without reckoning with the prolonged history of social and economic strife, further normalizing a world in which people are detached from the implications of inanimate objects. I therefore believe that an ethical approach must center the human so that we can properly understand the social issues that motivate technological ones.

**Reply by abi-200** [reply]
The personification of technology that you bring up is not something that I would have thought of on my own, but I do agree that this week's reading helped me center my own thoughts on where I stand with how much I use/depend on technology, even when the best solution would be not to do so. I think our society's reverence of technology has definitely crept in on us, because while it can and does simplify our lives and help us out at times, there is definitely a line to be drawn between it merely being a tool that we use in conjunction with social aspects and us being dependent on it solely. I agree that centering the human can help us dial back where needed as a result.

**Reply by perezsara59** [reply]
I think you bring up a really good point about the environmental impact of technology. Recently i've heard a lot more people talk about NFT's and since its all digital many people dont think it has any kind of environmental impact but I have recently learned that it actually does. While I still dont know much about this, I think its important for us as users/consumers to consider looking into the impacts before delving into things, particularly tech, that might not turn out to be as ethical as we thought because we cant rely on companies and corporations to always be honest about their ethical practices.

**Reply by luigimagdamit** [reply]
I really like your point on personifying technology. I think it is important to keep those that abuse it accountable. Another commenter discusses exploring the impacts of tech before we implement it. However, while I completely agree, I do not have confidence in the powers that be. If there is potential for abuse of technology, I believe that someone will abuse it for short term gain.

**Reply by emilysuh** [reply]
I appreciate your point about how popular discourse on technology makes technology seem omnipresent while disempowering people. Your suggestion to center people would emphasize that we as human beings have agency and the ability to change and alter our world, in spite of hegemonic forces like capitalism and modern nationalism. I also appreciate that you bring up the environment -- the planet too must be centered when we consider the ethics of technology, which would help us move away from the anthropocene approach to technology! :)

**Comment by cynthinee** [comment]
I haven’t seen The Social Network yet, mostly because I don’t have Netflix. I always find it interesting where people draw the line on what technologies to engage with and which ones to avoid. But I still have a Facebook account that I barely use, which I keep knowing that Facebook is terrible for users. However, my family really prefers it to keep in contact, so I just use it sparingly. 

I was thinking about Facebook and privacy though because I think it’s interesting how our most pressing concerns about privacy have really shifted over the years. I was on Facebook when it first rolled out to only college students in 2004 or 2005. In 2006, Facebook dramatically changed by introducing the News Feed, which is literally how all social media platforms work today. (Just chronological News Feed, this wasn't even the algorithm for ranking what gets shown on individual News Feeds.)  Back then, people, myself included, were like, "Ew we don’t want our friends noticing every single time we change something on our profile." I guess it was a little more akin to regular life where you went to certain profiles to check in on individual friends, not getting this overwhelming look into what everyone was doing or what the most frequent engagers were up to.  

Now news feed-esque life is pretty standard on social media, and I don’t think it’s a controversial privacy issue these days. 2006 wasn’t that long ago! I wonder what other practices we engage in right now that seem icky but years down the line, they’ll become standard practice that we don’t think twice about. 

For GIS, I think about what kinds of information we’re asking of our participants and how to ensure their privacy. There’s a lot that we can figure out based on various data points, and I wouldn’t want to put someone in a situation where they could be easily located. I’m also the type of person who looks at metadata in shared files (I think it’s a librarian thing), so I think that causes some paranoia about making sure people know what they’re sharing exactly. While it is extremely convenient for mapping purposes, do participants want to automatically have GPS location encoded into the photos they take on their phone? Maybe part of critical cartography and participatory mapping includes some kind of data or GIS literacy in which we impart some of our knowledge in how to keep communities safe onto those with whom we’re working. 

Speaking of geotags on photos, I definitely was the type of person who, several years ago, just had that feature turned off. I semi-recently turned it on, but I typically will disable sharing location when I share photos with others. But recently, I’ve noticed that I haven’t even been doing that that often anymore. I find it really difficult to sustain personal ethics around technology sometimes.

**Reply by abi-200** [reply]
It was interesting reading your point about how different social media was in 2006 - I didn't start using social media until later in high school around 2017, so for me the news feed-esque portion of social media that you mention is all I have ever known, and privacy concerns over social media have been consistently hammered into me for as long as I could remember. I also have an inactive Facebook that I use only for college and family purposes, but on other applications, I am definitely more relaxed when it comes to sharing locations. In fact, in spite of privacy concerns, I do have my location constantly shared with a couple of trusted friends because it actually makes me feel more secure that way, to have someone always know where I am.

Ensuring participant privacy even with aggregate data is a slippery slope because as you have said, there is a lot that can be found from just a single data point, and sometimes the level of information we can piece together is concerning and does not require technology that is too advanced/inaccessible to much of the public. I've been part of a a research project before with aggregate data collection, but myself and a couple of other researchers had secure access to individual files and information, so regardless of how the data is used, there is always the issue of single data points being collected and seen in some capacity.

**Reply by eandrusenki** [reply]
Like Abita mentioned I definitely was not on social media in 2006 (I was 6 years old haha), to me social media wasn't an everyday normal thing until about 2014. But I think it is interesting you mention how social media was viewed back then versus now and what things we might be uncomfortable with now and maybe will be in the future. There are always ideas surrounding privacy and what should or should not be a thing. Before, it was taboo to post pictures of your newborn babies online and now it happens all the time. In pictures you can post your location on Instagram, heck on snapchat if you choose all your friends can see where you are all the time.
Because I was adopted, my mom had a fear of my birth parents finding out where I was and stealing me ALL the time and so for me, privacy was extremely important and I value participants privacy, especially with my project where the topic is sensitive.

**Reply by edmondywen** [reply]
I never really thought about how there are so many implicit privacy agreements that people acquiesce to by simply using certain pieces of technology like geotags on photos. While there is a benefit of convenience for some, the fact that people essentially opt in by default to these privacy sensitive features really highlights how ethical lines have rapidly blurred in the 21st century. I love your point about GIS literacy because a lot of people, including myself, never really thought about the full impact of metadata.

**Reply by ronnie-blanche** [reply]
Hey, Cynthia! I really enjoyed your anecdote on how Facebook was like in 2006 and your own qualms with it. I don't use it a lot anymore, but I definitely relate to feeling annoyed every time I change something on my profile and it's announced to all of my friends. For example, I still have to change the notification settings to private every time I change my profile picture. It's supper interesting hearing how you had the same issue despite our experiences having occurred at different times. I guess a pro and con to technology keeping everyone connected is how transparent everything can be. Which then raises the question: is being so connected a good thing or bad thing?

**Comment by eandrusenki** [comment]
I believe that technology and technological advances are incredible and have helped society come such a long way, however, I think there is a shift from helping society and people to prioritizing technology and that's where our focus has moved to. It is taking a lot away from humans and society as a whole. In a perfect world obviously, technology would be used for good, but that is not the case and I think it has turned into people viewing other people as numbers and things rather than people with emotions.
This is especially true when it comes to technology and the way it can be used to break privacy and for surveillance. One example I always think about when talking about ethics and technology is predictive policing. This idea that if you live in a neighborhood you could be a criminal, if you do x, y, z we will label you as a criminal. I think it is one of the worst ways technology is used because it dehumanizes society so much. 
My biggest issue with technology is society's ease with just blaming technology for a myriad of our problems. While it may be correct that technology causes many problems, we have to remember who built and coded and programmed technology. That members of society have implicit biases which are passed on to technology. That capitalism, materialism, biases, all effects what WE choose to do with technology ,

**Reply by edmondywen** [reply]
Predictive policing is an excellent example that highlights the moral problems infused into human usage of technology since it utilizes many "big data" tools that tend to reduce individuals to numbers and statistics on a chart. It also relates to the example in the reading about how some people are forced to pay higher insurance costs based simply on where they live, not their personal experience. Overall, there appears to be a trend of dehumanization as the human being becomes separated by a screen.

**Reply by cynthinee** [reply]
I've been really concerned with the ways technology has been integrated into policing. In Long Beach, we have this really ominous police mobile unit called [MobilEye](https://www.presstelegram.com/2015/08/02/new-long-beach-police-mobile-unit-keeps-watch-in-high-crime-areas/), which is an armored car with surveillance cameras that sits in areas that 'need it the most,' which inevitably end up in very specific neighborhoods in Long Beach. While the motto of the MobilEye is "Watching out for the neighborhood," it very much feels like it only polices low-income communities and communities of color. When I go on walks, it definitely has the opposite effect on me: I don't feel safe, and I wonder what data is, in that moment, being collected on me and where that information will go and be used.

**Reply by luigimagdamit** [reply]
I think your second take is very interesting, saying the existence of our fundamental issues affects our use of technology. And I really agree. While revolutionary technology such as machine learning exists, the prevalence of capitalism will see the abuse of this technology so long as it can guarantee its abusers some marginal gain.

**Reply by perezsara59** [reply]
I really agree with what you said especially the part of implicit bias being passed on to technology. A lot of times these AI programs, especially with policing, are conveyed as being neutral which just isn't true, previous class readings have also touched on how we all have a bias. I remember there was a CA ballot proposition to end cash bail but it was gonna use some type of AI to conduct a risk assessment (https://ballotpedia.org/California_Proposition_25,_Replace_Cash_Bail_with_Risk_Assessments_Referendum_(2020) ). Many people who wanted to end cash bail were weary of this new system and its possibly bias.

**Reply by ronnie-blanche** [reply]
I really appreciated how you pointed out that our problems with technology actually arise because of the creators and how they benefit from the social institutions we have in place. We often see technology as a resource to help improve and connect communities as that's what it was intended for. However, many people often forget that systems of oppression work in many different ways and definitely influence how resources are used and created. Just because something was meant for doing good does not mean it has to be only used for that, and it is up to us to dismantle these systems and ensure that technology is being used to benefit everyone and not just certain populations.

**Comment by luigimagdamit** [comment]
In my opinion, I believe that society and technology should be seen through a lens of democratization. I think that one of the most comforting trends in modern history is the development of more accessible information and tools. For example, anyone with a computer right now could create digital art, videos or music because the tools exist and are readily accessible to the common people. You can even make music on your phone, something that was completely unimaginable half a century ago. 

Anyways, tools like these and the Internet, I believe, really have the potential to empower the common person since it gives the average person more independence and freedom in developing their messages or art. In this case, we see both tech and society helping each other. I believe this is an adequate display of constructivism between both technology and society. 

However, despite these advantages, the internet and technology do have their deterrents on our collective wellbeing. The influx of misinformation, surveillance, the treatment of users as data points and predatory time wasters exist as a dark counterpoint to empowerment through technology.

**Reply by abi-200** [reply]
Your point about the lens of democratization and technology being a comfort definitely rings true to me, because with increased access comes increased potential. However, it is important to bring up (in my opinion) that just increasing access does not in and of itself alleviate social inequities when it comes to data use - for example, just because a technology is available, does not mean that disadvantaged communities will have the time and/or training to dedicate themselves to it, due to other social factors beyond their control (ex. have to allocate time elsewhere). I do agree that the state of access and democratization now is better than it has been in years past, and I also agree with your point about technology and society needing to help each other too.

**Reply by cynthinee** [reply]
I agree that technology and democratization go hand in hand. I love to see how the internet has enabled such amazing interactions, creativity, and change, which is really important to remember. In my field, we talk about information cynicism in relation to how the internet has also been a source of bad information or straight up misinformation: the idea that people feel so powerless to actually find reliable and useful sources amidst all the utter crap out there. Remaining hopeful and seeing the power and utility in technology is still a really important strategy towards making lasting change.

**Reply by hrfrahn** [reply]
I liked your pint about the democratic/emancipatory potential of technology; it's hard to argue that technology has no good aspects when we're conducting this entire class through Zoom/Github. However, it does seem like an overreliance on large tech companies has the potential for anti-democratic effects.

**Comment by ronnie-blanche** [comment]
How far technology has come and the plethora of resources we now have at our disposal thanks to technology has bettered society in a plethora of manners, however technology has definitely become a conduit through which people can also do more harm than good. Technology can be easily weaponized to further separate people, as well as spread false information and turn people against each other. Though people tend to blame the resources themselves for these issues, we as users must realize that it is the actions WE take that creates problems and not the technology itself. Change cannot be enacted if we don't step back and own up to our own mistakes.
An issue that I personally have is how privacy and personal information can easily be tracked or sold. A lot of people believe that this issue is because of how we use technology/our reliance on it – which is not the case. These problems arise because of the ethics of users and creators, often people in power. These people have little to no concern about ethics because of : capitalism, white supremacy, sexism, etc. People at the top of the power benefit from these institutions and are more concerned with benefiting themselves rather helping improve or change society for the better.

**Reply by emilysuh** [reply]
I appreciate your point about power and structures impacting how technology is used! I agree that we need to be critical about how technology harms society and that we must take ownership of how technology gets used. I wonder what people might need in order to feel empowered and comfortable in pushing back against technology elites in this way. One of the first things that comes to mind is increasing people's overall technology literacy so they can really understand what they are using and how it is being used.

**Comment by emilysuh** [comment]
We must critically contend with technologies and how they are enmeshed with existing power structures, scrutinizing whether or not the technologies uphold, facilitate or strengthen systemic oppression and social inequalities. Under global capitalism, new technologies are commodified and marketed with profit as the primary goal, rather than social good. Because a small elite class often owns exclusive rights to produce and market new technologies, there is a huge disparity in power as it relates to technology and how it is used and accessed. Because quick profit for a small group of technocrats is the goal, we see the creation of more expensive, flashier technology that a limited few can access, rather than inexpensive, sustainable technology (or the re-use or re-purposing of existing technology such as quarantine protocol implementation) that can be widely and affordably accessed. Sometimes, even when the technology exists, the problem at hand is not solved because of the social inequities that have failed to be addressed. As an example, PrEP is 99 percent effective at preventing people from contracting HIV; this medical technology has been around since 2012. However, epidemiologists have yet to see a population-level impact of PrEP and less than one in five who are indicated to take PrEP actually take it. This is because the structural, social aspects of implementing and popularizing this technology have not been fully addressed: health care access, homophobia and transphobia, and racism, among others. These aspects of social inequality must be centered in order for any technology to be implemented in a way that benefits and empowers the impacted population.

**Reply by emilysuh** [reply]
For more on PrEP & how social inequalities affect its implementation: https://prevention.nih.gov/about-odp/directors-messages/2020/qa-dr-julia-marcus-2020-odp-early-stage-investigator-lecture-awardee

**Reply by hrfrahn** [reply]
PrEP is a really good (and somewhat depressing) example of how new technology has to be developed/implemented with equity and society in mind from the start to have real effects. Personally, I'd heard of PrEP, but never realized just how effective it was or how much of a usage gap exists in people who are indicated to take it.

**Comment by perezsara59** [comment]
I actually watched _The Social Dilemma_ on Netflix and I remember how shocked I was at the fact that they were disclosing things that were just confirming thoughts I had previously had. As a consumer of technology I obviously want my privacy protected and dont want my data being sold. Capitalizing off of people and the way they use their devices is unethical to me, especially when people are unaware. This experience as a consumer influences me when I use technology (i.e. the group assignment in this class) to consider how to protect the user while allowing them to share their experience. I do think its important to remember who is designing the tech, humans who are also a part of society, contribute to these projects. Technically a lot of this information is probably included in the fine print which many dont take the time to read, but when companies project a safe environment consumers dont think twice, they trust them. As a society though we tend to spread things that sound convincing which may not be true, its easy for some to say "well just fact check it!" but with so much information available, how can someone who is easily influenced/impressionable decide what is true and isn't true? 

Under capitalism efficiency is important to meet demand. Companies are opting for robots to build things because they dont require a livings wage, healthcare, benefits, and dont need to take breaks. You can argue that they're being ethical because no humans are being exploited but, personally, leavings thousands of people unemployed is not ethical. Humans need money to survive under capitalism and its becoming increasingly more difficult for people to guarantee a living wage. Then again, we as a society drive the need/demand for supply.

**Reply by ramirezirvin** [reply]
Yeah, I recently found out about one's personal information being sold online, it is a very scary thing to witness. I have read a post regarding this, where a person has sent information with another close individual through WhatsApp and the company itself has taken that information without the user's consent and selling it to companies abroad. I would never imagine a messaging app could do that, which makes me scared of what I send within WhatsApp now and try sticking to regular text messages instead of apps. Technology can be a scary thing if used wrong for sure.

**Comment by hrfrahn** [comment]
I believe that technology on its own can best be described as "ethically neutral". It's not inherently good or bad; what determines its ethical value is how it is used. There are undeniable examples of technology being used democratically and beneficially (think of how Wikipedia allows instant access to crowdsourced, mostly reliable information), but there are also countless examples of technology being co-opted by non-democratic forces (for example, large tech companies that only care about profitability). Under capitalism, the market often drives technology to be used in undemocratic ways in order to maximize profit. 

If technology itself is neither good nor bad, it's up to the designers of any specific new technology to ensure that their implementation serves civil society and works in favor of democratizing information rather than against it. A footnote from the Crampton reading really stood out to me: 

> all maps are politicized in some manner; it is just that their politics does not challenge the status quo.

This principle can be applied not just to maps/GIS, but all forms of technology/media; for example, making an app like Facebook that relies on the commodification of user data might not seem as political as making one that completely rejects tracking, but this is only because "surveillance capitalism" has become so normalized that we regard it as the only proper way to structure our social media.

**Reply by ramirezirvin** [reply]
Yes I totally agree, technology could be described as ethically neutral depending on the usage. There is so much going on within technology that sometimes it is neither a positive nor a negative thing.  It is certainly up to the designers as you mentioned to ensure how it is used in society, and there no way of knowing how it can be used within those individuals in the end.

**Comment by ramirezirvin** [comment]
In all honestly, technology can be used for good purposes and for bad, as there are also technological trends in society.
When the internet first came around, it was something revolutionary that paved the way to new jobs and resources once it became public. It is still used in positive ways, but also bad such as hacking for malicious purposes. Though another technological advancements that has been made are cars, televisions, smart houses, gaming systems, music software, video and photo editing some are used to help facilitate one's life, others for fun, leisure, or work-based. Though policing what people do with the technological access they have is out of one's hands if they choose to disclose sensitive information or attack a company or an individual through hacking or a virus it is kind of difficult, also something in a more drastic view, stealing one's work and claiming it as their own, which is commonly seen on the internet. Sometime's the villains are just normal day individuals who use social media for negative use, and that is where society can also use technology for bad or good. 
In the end it is hard policing each individual with what they can do with technology, but it sure does has its pros and cons.
